AppCenter/AppCenter/Internals/Channel/MSChannelGroupDefault.m

```diff
@@ -27,7 +27,7 @@ - (instancetype)initWithIngestion:(nullable MSAppCenterIngestion *)ingestion {
     _storage = [MSLogDBStorage new];
     if (ingestion) {
       _ingestion = ingestion;
-      _ingestion.authToken = [MSAuthTokenContext sharedInstance].authToken;
+      _ingestion.authToken = [[MSAuthTokenContext sharedInstance] authToken];
```

Comment:
If we want to still use the `authToken` as a property we can declare it as readonly property in the "MSAuthTokenContext.h" but leave it as read/write in "MSAuthTokenContextPrivate.h".